src/settings/FundTypeSettings/FundTypeSettings.js

```diff
@@ -40,6 +42,11 @@ const FundTypeSettings = ({
       hiddenFields={hiddenFields}
       nameKey="name"
       id="fundTypes"
+      canCreate={hasEditPerms}
+      actionSuppressor={{
+        edit: () => !hasEditPerms,
+        delete: () => !hasEditPerms,
```

Comment:
you should frame the const hasEditPerms based on the specific PUT permission.
make another declaration for hasDeletePerms based on the specific DELET permission.
hook them accordingly to the actionSuppressor